### ISSUE TITLE
How can I generate api-doc for query with model like RequestBody() ?

### ISSUE THREAD

**Issue by wizard-x** [nelmio/NelmioApiDocBundle#2080]
Today I have to use a textarea for repesenting and editing model properties as query params, and I have no access to view the schema..
But RequestBody() looks like the same and has an example of values and the schema..

Something like
```
    #[OA\Parameter(
        name: 'query',
        in: 'query',
        schema: new OA\Schema(
            ref: new Model(
                type: SomeQueryRequest::class,

```

And RequestBody can be declared like this

```
    #[OA\RequestBody(
        content: new OA\JsonContent(
            ref: new Model(type: SomePostRequest::class),
        ),
    )]
```
Is there any possible way to do that for the model passtrought query parameters in the project?

I have to look something like 
```
    #[OA\QueryParameter(
        content: new OA\JsonContent(
            ref: new Model(type: SomePostRequest::class),
        ),
    )]
```
Or similar

Thanks

**Comment by mario-fehr** [comment]
At the moment, only with a workaround found here: #1721

**Comment by mario-fehr** [comment]
@GuilhemN any change we get this error fixed? #issuecomment-1128535488

**Comment by wizard-x** [comment]
It's sad (
Sorry for duplicate question
